### PR TITLE
request-bottle: permit dispatching against taps

### PR DESCRIPTION
**Pull request by jonchang**
This should let us build bottles for e.g. xorg formulae.

See https://github.com/Linuxbrew/homebrew-xorg/pull/687

### CODE REVIEW

File: cmd/request-bottle.rb [hunk 1]
```diff
@@ -59,6 +63,14 @@ def request_bottle
     odie "Email not specified" if email.empty?
 
     args.named.to_resolved_formulae.each do |formula|
+      # Always dispatch core formulae against Homebrew/linuxbrew-core,
+      # even on macOS.
+      remote = if formula.tap == CoreTap.instance
```
Comment (dawidd6) [inline cmd/request-bottle.rb:68]: Wouldn't `formula.tap.full_name` be sufficient?

Comment (jonchang) [reply cmd/request-bottle.rb:68]: We need to dispatch against the remote of the tap, which might be different from the filesystem path.

Comment (dawidd6) [reply cmd/request-bottle.rb:68]: Umm, I still don't get it, but I trust you :smile:

Comment (iMichka) [reply cmd/request-bottle.rb:68]: Because the filesystem path can be homebrew/homebrew-core and the remote path homebrew/linuxbrew-core for example.

**Review by iMichka — APPROVED** [review]

**Pull request merged**